api/commands/reminders/remind.js

```diff
@@ -52,29 +53,30 @@ module.exports = class RemindCommand extends Command {
     if (!chrono.parseDate(datetime)) {
       return msg.say(exceptions.invalid_datetime_format);
     }
-    
-    let scheduledTime = moment(chrono.parseDate(datetime)).calendar(
-      moment.now(), "M/D/YYYY h:mm a");
 
-    let millisecondsTillReminder = chrono.parseDate(datetime).getTime() -
-      moment().valueOf();
-
-    if (millisecondsTillReminder < 0) {
+    if (moment(chrono.parseDate(datetime)).diff(moment()) < 0) {
       return msg.say(exceptions.past_time);
     }
 
     return selectTz([target.username, target.discriminator])
-    .then(res => {
-      if (!res) {
+    .then(timezone => {
+      if (!timezone) {
         return msg.say(exceptions.timezone_not_set);
       } else {
+        let timeWithOffset = parseDate(datetime, timezone);
+
+        let formattedTime = moment(chrono.parseDate(timeWithOffset)).calendar(

```

Comment:
At first glance this appears to be a bit circular
parseDate: `chrono.parse` -> add offset -> returns `moment.offset.toISO`
that is then wrapped in
`moment` -> `chrono.parseDate`

is there a way to condense this so that it only needs to use chronos/moment at most once each?